packages/ai-ollama/src/node/ollama-language-model.ts

```diff
@@ -77,12 +88,14 @@ export class OllamaModel implements LanguageModel {
     }
 
     protected async handleStructuredOutputRequest(ollama: Ollama, request: LanguageModelRequest): Promise<LanguageModelParsedResponse> {
+        const settings = this.getSettings(request);
         const result = await ollama.chat({
+            ...settings,
             ...this.DEFAULT_REQUEST_SETTINGS,
             model: this.model,
             messages: request.messages.map(this.toOllamaMessage),
             format: 'json',
-            ...request.settings
+            stream: false,
```

Comment:
Do we never stream on purpose here?

Reply:
That was the behavior before. We have add this explicitly, otherwise we have a type error because the function of the ollama API is overloaded. It does make sense to not stream structured output.

Reply:
If you like you can comment in the code why this property was set

Reply:
@dhuebner Can you confirm that this is intended?

Reply:
@JonasHelming 
I never received structured requests when using Theia AI so I can't really check if it works or not. But I think streams are useful mostly in chat context.